_studio/shared/umc/codec/color_space_converter/src/umc_deinterlacing.cpp

```diff
@@ -138,8 +138,8 @@ static void DeinterlacingEdgeDetect(uint8_t *psrc,
 
   mfxiCopy_8u_C1R(psrc, 2*iSrcPitch, pdst, 2*iDstPitch, roi);
 
-  memcpy_s(pdst + iDstPitch, iSrcPitch, psrc + iSrcPitch, iSrcPitch);
-  memcpy_s(pdst + (h - 1)*iDstPitch, w, psrc + (h - 2) * iSrcPitch, w);
+  std::copy(psrc + iSrcPitch, psrc + 2*iSrcPitch, pdst + iDstPitch);
```

Comment:
You bring a big code to open source. I believe you need to bring it "fixed". So, please:
1. Take this patch offline to the closed source version you are open sourcing
2. Bring fixed code initially. In a way this patch should be squashed to one of the previous.